plugin.go

```diff
@@ -55,11 +58,50 @@ func (p *Plugin) Init(cfg Configurer) error {
 
 	p.prop = propagation.NewCompositeTextMapPropagator(propagation.TraceContext{}, propagation.Baggage{}, jprop.Jaeger{})
 
+	// Configure CORS options
+	if p.cfg.CORS != nil {
+		corsOptions := cors.Options{
```

Comment:
You may create a pointer to a `cors.Options` while declaring them, like `p.cors = &cors.Options{...}`

Reply:
Updated. Additionally, added a possibility to configure options response status code `OptionsSuccessStatus` (defaults to 200 as was before)